packages/lib/server/eventTypeSelect.ts

```diff
@@ -50,26 +50,4 @@ export const eventTypeSelect = Prisma.validator<Prisma.EventTypeSelect>()({
   secondaryEmailId: true,
   bookingLimits: true,
   durationLimits: true,
-  parent: {
-    select: {
-      team: {
-        select: {
-          id: true,
-          members: {
-            select: {
-              user: {
-                select: {
-                  name: true,
-                  username: true,
-                  email: true,
-                },
-              },
-              role: true,
-              accepted: true,
-            },
-          },
-        },
-      },
-    },
-  },
```

Comment:
Too much nesting here, was slowing down queries considerably for organization with a lot of members @vijayraghav-io